the 4th argument to add_metadata_gen_target defines the NAME.h of the output.
metadata.h is already used by the build- this conflicts

having a duplicate output name is invalid. ninja file spec enforces this and
fails

diff --git a/cpp/CMakeLists.txt b/cpp/CMakeLists.txt
index d2d111d..cdd7a08 100644
--- a/cpp/CMakeLists.txt
+++ b/cpp/CMakeLists.txt
@@ -394,7 +394,7 @@ add_metadata_gen_target (
   ${TEST_METADATA_TARGET}
   "${RESOURCES_DIR}/PhoneNumberMetadataForTesting.xml"
   "test_metadata"
-  "metadata"
+  "test_metadata"
 )
 list (APPEND TESTING_LIBRARY_SOURCES "src/phonenumbers/test_metadata.cc")
 
